shell/app/common/__tests__/components/__snapshots__/loading-skeleton.test.tsx.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Is this snap file useful?

Reply:
this component without events and data，the rendering result is just a Dom tree